docs/src/docs/asciidoc/guides/how-to-custom-claims-authorities.adoc

```diff
@@ -0,0 +1,49 @@
+[[how-to-extension-grant-type]]
+= How-to: Add authorities to custom claims in JWT-based access tokens
+:index-link: ../how-to.html
+:docs-dir: ..
+:examples-dir: {docs-dir}/examples
+
+This guide demonstrates how to append resource owner authorities to a JSON Web Token (JWT)-based access token.
+The term 'authorities' may be described in varying forms such as roles, permissions, or groups of the resource owner.
```

Comment:
Use double quotes for 'authorities'

may be described in varying forms -> may represent varying forms